targets/TARGET_NXP/scripts/LPC.py

```diff
@@ -47,6 +53,14 @@ def is_patched(bin_path):
 
 
 if __name__ == "__main__":
-    binary = sys.argv[1]
+    artefacts_location = sys.argv[1]
+
+    try:
+        binary = list(pathlib.Path(artefacts_location).glob("*.bin"))[0]
```

Comment:
Is this assuming there is just one binary ? Shall this be documented here (the assumption made)

Reply:
Yes that is the assumption made.
@ifyall is this a fair assumption?